onnx/numpy_helper.py

```diff
@@ -874,3 +704,14 @@ def create_random_int(
         return np.random.randint(start, end, size=input_shape).astype(dtype)
     else:
         raise TypeError(f"{dtype} is not supported by create_random_int.")
+
+
+def saturating_cast(x: np.ndarray, dtype: np.dtype) -> np.ndarray:
```

Comment:
@yuanyao-nv Does saturating_cast look good to you?

Reply:
Should be ok.